akka-remote/src/test/scala/akka/remote/serialization/DaemonMsgCreateSerializerSpec.scala

```diff
@@ -160,26 +185,47 @@ class DaemonMsgCreateSerializerSpec extends AkkaSpec {
       }
     }
 
-    def verifySerialization(msg: DaemonMsgCreate): Unit = {
-      assertDaemonMsgCreate(msg, ser.deserialize(ser.serialize(msg).get, classOf[DaemonMsgCreate]).get.asInstanceOf[DaemonMsgCreate])
-    }
+  }
+}
+
+class DaemonMsgCreateSerializerNoJavaSerializationSpec extends AkkaSpec(
+  """
+   akka.actor.allow-java-serialization=off
```

Comment:
shall we try to use `akka.actor.allow-java-serialization=off` in some of the remote router tests?